src/main/kotlin/racingcar/CheckException.kt

```diff
@@ -0,0 +1,36 @@
+package racingcar
+
+class CheckException {
+
+    fun firstInput(car: String) {
+        if (car == "" || !car.contains(",")) { // 빈문자열 또는 자동차 이름 하나만 입력 시
+            throw IllegalArgumentException()
```

Comment:
```suggestion
            throw IllegalArgumentException("자동차 이름은 하나 혹은 빈 문자열일 수 없습니다.")
```
코드를 보면 주석을 통해 어떤 오류가 발생 했는지 알 수 있지만, 빌드를 하고 실행만 하는 입장에서는 어느 부분에서 입력이 잘못 됐는지 알기 힘들 것 같습니다.
오류 메세지를 표기해주는 것도 좋은 방법인 것 같아요!

Reply:
엇 저번 주차에서는 추가 했었는데 이번 과제 하면서 적절한 에러 메세지를 고민하다가 빠트렸네요...ㅜㅜ 좋은 지적 감사합니다!